public_dropin_gpu_environments/nemollm_inference_ms/dr_requirements.txt

```diff
@@ -4,6 +4,10 @@
 #
 #    pip-compile --index-url=https://artifactory.devinfra.drdev.io/artifactory/api/pypi/pypi/simple/ --no-emit-index-url --no-emit-trusted-host dr_requirements.in
```

Comment:
must not expose our internal artifactory

Reply:
removed